lib/src/omise_http_client.dart

```diff
@@ -22,13 +23,15 @@ class OmiseHttpClient {
   /// is thrown. The [httpClient] is used to make HTTP requests. If not provided,
   /// a default [http.Client] is used.
   OmiseHttpClient({
-    required this.publicKey,
-    required this.secretKey,
+    this.publicKey,
+    this.secretKey,
     this.userAgent,
     http.Client? httpClient,
     this.enableDebug = false,
+    this.ignoreNullKeys = true,
```

Comment:
Can you please tell me more about this `ignoreNullKeys`?

Reply:
When you use the models to create a request and the models have some optional fields, lets say the request model has the `age` field as optional, so when the request is sent the json will include `age:null`. The `ignoreNullKeys ` allows us to control whether to send keys that have null values or remove them.